File: hh-front/src/models/plan_cuenta_model.js
import { UsuarioModel } from './usuario_model'

export class PlanCuentaModel {
  constructor () {
    this.id = ''
    this.nombre = ''

    this.creador = new UsuarioModel()
    this.creada = ''
    this.modificador = new UsuarioModel()
    this.modificada = ''
    this.eliminador = new UsuarioModel()
    this.elimianda = ''
  }
}

File: hh-back/src/main/java/gloit/hiperionida/helios/model/PlanCuentaModel.java
package gloit.hiperionida.helios.model;

import gloit.hiperionida.helios.util.model.AbsAuditoriaModel;

import jakarta.persistence.*;
import lombok.*;
import lombok.experimental.SuperBuilder;

import java.time.LocalDateTime;

@EqualsAndHashCode(callSuper = false)
@Entity
@Getter
@Setter
@SuperBuilder
@Table(name = "plan_cuenta")
public class PlanCuentaModel extends AbsAuditoriaModel {
    private String nombre;
    @Column(columnDefinition = "TEXT")
    private String notas;

    public PlanCuentaModel() {
    }

    public PlanCuentaModel(Long id, Long creadorId, LocalDateTime creada, Long modificadorId, LocalDateTime modificada, Long eliminadorId, LocalDateTime eliminada, String nombre, String notas) {
        super(id, creadorId, creada, modificadorId, modificada, eliminadorId, eliminada);
        this.nombre = nombre;
        this.notas = notas;
    }
}
